xml/System.ComponentModel.DataAnnotations/ValidationAttribute.xml

```diff
@@ -93,7 +93,15 @@
       <Parameters />
       <Docs>
         <summary>Initializes a new instance of the <see cref="T:System.ComponentModel.DataAnnotations.ValidationAttribute" /> class.</summary>
-        <remarks>For more information about this API, see &lt;see href="/dotnet/fundamentals/runtime-libraries/system-componentmodel-dataannotations-validationattribute--ctor"&gt;Supplemental API remarks for System.ComponentModel.DataAnnotations.ValidationAttribute..ctor&lt;/see&gt;.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Note to inheritors
+
```

Comment:
NIT: this doesn't have `<note type="inheritinfo">` https://source.dot.net/#System.ComponentModel.Annotations/System/ComponentModel/DataAnnotations/ValidationAttribute.cs,41
```suggestion

```